.github/ISSUE_TEMPLATE/good_first_issue.md

```diff
@@ -0,0 +1,11 @@
+The purpose of the good first issue label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.
+
+Useful skills:
+
+(For example “C++11 std::thread”, “Qt5 GUI and async GUI design” or “basic understanding of Bitcoin mining and the Bitcoin Core RPC interface”)
+
+Want to work on this issue?
+
+You do not need to request permission to start working on this. It is encouraged (but not compulsory) to comment on the issue if you are planning to work on it. This will help the maintainers of the project monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it.
```

Comment:
> It is encouraged (but not compulsory) to comment on the issue if you are planning to work on it. This will help the maintainers of the project monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it.

I don't think it's true that maintainers are actively monitoring which new contributors are working on which up for grabs issues. I'd just replace this with "You may wish to comment on this issue if you start working on it. You can also report progress or ask for help here."

Reply:
I think it helps to encourage people to report in progress/issues more frequently so that we have less "XY, are you still working on this?".

Reply:
Yeah that was my thinking @fjahr. Even if it isn't strictly true (as @jnewbery says) that maintainers actively monitor this I still think it encourages the sought after behavior, namely let everyone (maintainers, author of issue, other new contributors) know you are planning to work on this and keep everyone updated with your progress. Not compulsory but actively encouraged.

Reply:
I'll keep this as it is unless @jnewbery comes back with strong reservations.

Reply:
My reservation is you're telling people that maintainers are monitoring this, which isn't true!

Reply:
Haha ok fair enough. How about replacing "the maintainers" with "other contributors"?

"This will help other contributors monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it."

Reply:
was just going to suggest something similar: s/This will help the maintainers of the project monitor.../This will help others interested in contributing to see.../